css3d.js

```diff
@@ -23,6 +23,7 @@ css3d = function (selector) {
     this.getMatrix(); // set an initial value for the matrix
 };
 
+css3d.touchIsSupported = typeof window.TouchEvent !== "undefined";
```

Comment:
I like the idea of calculating this book just once and also exposing it publicly. Are we sure TouchEvent is not defined in normal browsers? Seems weird to me. If so can you post a reference here and preferably add a comment in code?

Reply:
As of now, `TouchEvent` is defined in the desktop version of chrome, but not in safari or Firefox.
As I understand, Firefox interpretes touches as mouse events (at least on Windows 10), while Chrome "does it right" and interpretes touches as Touch Events and mouse clicks as Mouse Events.
I'll add a comment the line.

Reply:
I also like the idea of calculating this once, but it could easily be overridden. A function could always return the correct bool.
```suggestion
const touchIsSupported = typeof window.TouchEvent !== "undefined";
css3d.touchIsSupported = () => touchIsSupported;
```

Reply:
@lightningboss hmmm aber die Funktion könnte auch überschrieben werden. oder meinst du was anderes?

Reply:
@lightningboss juckt I think. If someone wants to overwrite it I say go for it :P We could also do an `Object.freeze` on `css3d` but I think it's fine as is for now.